packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -269,12 +291,15 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     });
 
     this.paymentChannelClient.onChannelUpdated(async (channelState: ChannelState) => {
-      const {pseChannelId, peerChannelId, peerAccount} = wire.paidStreamingExtension;
+      const {seedingChannelId, leechingChannelId, peerAccount} = wire.paidStreamingExtension;
 
-      if (channelState.channelId === pseChannelId || channelState.channelId === peerChannelId) {
+      if (
+        channelState.channelId === seedingChannelId ||
+        channelState.channelId === leechingChannelId
+      ) {
         if (channelState.status === 'closed' || channelState.status === 'closing') {
-          wire.paidStreamingExtension.pseChannelId = null;
-          wire.paidStreamingExtension.peerChannelId = null;
+          wire.paidStreamingExtension.seedingChannelId = null;
+          wire.paidStreamingExtension.leechingChannelId = null;
```

Comment:
I just noticed that I made a mistake, in this if statement. 
This might make problems: There is the posibility that a peer has both a seedingChannel and a leechingChannel, and only one of them got closed (for example, if it finished downloading the file). This if statement would remove both ids.

Reply:
It's not something that you introduced, I think I introduced it at some point

Reply:
I think it might be better to make an issue about this, instead of fixing it in this PR though

Reply:
https://github.com/statechannels/monorepo/issues/1911